Unable to start language server in VS Code workspace

Platform: macOS 14.4.1, M1 arm64
OPA version: 0.63.0
Regal version: 0.20.0
Client: VS Code: 1.88.0

I haven't been able to get the OPA extension for VS Code to work as intended and can't seem to find any issue in GitHub or Slack like this one. I'm not 100% sure if this is an extension issue or a Regal language server issue, but I'm leaning towards the latter so I'm opening the issue here to start.

Here is my full output from `regal-ls` upon opening a `.rego` file:
```
running in workspace mode
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x100c270c0]

goroutine 33 [running]:
github.com/styrainc/regal/internal/lsp.(*LanguageServer).loadWorkspaceContents.func1({0x14000040be7?, 0x3b?}, {0x0?, 0x0?}, {0x1013600f8, 0x140002d7380})
	/home/runner/work/regal/regal/internal/lsp/server.go:628 +0x40
path/filepath.WalkDir({0x14000040be7, 0x3b}, 0x1400004bc68)
	/home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.22.0.linux-amd64/src/path/filepath/path.go:531 +0xa4
github.com/styrainc/regal/internal/lsp.(*LanguageServer).loadWorkspaceContents(0x1400033e5b0)
	/home/runner/work/regal/regal/internal/lsp/server.go:626 +0x5c
github.com/styrainc/regal/internal/lsp.(*LanguageServer).handleInitialize(0x1400033e5b0, {0x1400027caa0?, 0x1721?}, 0x101200880?, 0x1400007c8a0)
	/home/runner/work/regal/regal/internal/lsp/server.go:606 +0x354
github.com/styrainc/regal/internal/lsp.(*LanguageServer).Handle(0x1400033e5b0, {0x101366e30, 0x1400031fb80}, 0x14000320870, 0x1400007c8a0)
	/home/runner/work/regal/regal/internal/lsp/server.go:90 +0x1a0
github.com/sourcegraph/jsonrpc2.(*HandlerWithErrorConfigurer).Handle(0x14000318b00, {0x101366e30, 0x1400031fb80}, 0x14000320870, 0x1400007c8a0)
	/home/runner/go/pkg/mod/github.com/sourcegraph/jsonrpc2@v0.2.0/handler_with_error.go:21 +0x4c
github.com/sourcegraph/jsonrpc2.(*Conn).readMessages(0x14000320870, {0x101366e30, 0x1400031fb80})
	/home/runner/go/pkg/mod/github.com/sourcegraph/jsonrpc2@v0.2.0/conn.go:205 +0x2a4
created by github.com/sourcegraph/jsonrpc2.NewConn in goroutine 1
	/home/runner/go/pkg/mod/github.com/sourcegraph/jsonrpc2@v0.2.0/conn.go:62 +0x1d0
[Error - 10:10:19 PM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 10:10:19 PM] Connection to server got closed. Server will not be restarted.
[Error - 10:10:19 PM] Regal LSP client client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 10:10:19 PM] Server process exited with code 2.
```

I tried a few different settings with no success, but this is what I currently have configured in VS Code:
```json
  "opa.dependency_paths.opa": "/opt/homebrew/bin/opa",
  "opa.dependency_paths.regal": "/opt/homebrew/bin/regal",
  "opa.languageServers": [
    "regal"
  ],
```

Please let me know if there's any additional information that I can provide or debug with.

--- a/internal/lsp/server.go
+++ b/internal/lsp/server.go
@@ -625,7 +625,7 @@ func (l *LanguageServer) loadWorkspaceContents() error {
 
 	err := filepath.WalkDir(workspaceRootPath, func(path string, d os.DirEntry, err error) error {
 		if err != nil {
-			return fmt.Errorf("failed to walk workspace dir %q: %w", d.Name(), err)
+			return fmt.Errorf("failed to walk workspace dir %q: %w", path, err)
 		}
 
 		// TODO(charlieegan3): make this configurable for things like .rq etc?

